### PR TITLE
Render Dragons List

### PR DESCRIPTION
**Pull request by mrsamsharma**
- Created a component which has basic list structure for Dragon Item
- Used `useEffect()` to fetch Data from Dragons API
- Used `useSelector()` to store data in API

### CODE REVIEW

File: src/components/DragonItem.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const dragonItem = (props) => {
+  /* eslint-disable */
+  const { name, type, img  } = props;
+  /* eslint-enable */
```
Comment (shella12) [inline src/components/DragonItem.js]: - [ ] I suggest not disabling the linters. Linters ensure you use best practices in programming. Try resolving linter errors manually

File: src/components/DragonItem.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+  return (
+    <>
+      <div className="dragon-item">
+        {/* eslint-disable */}
+        <img alt="DragonPicture" src={img} />
+        {/* eslint-enable */}
```
Comment (shella12) [inline src/components/DragonItem.js]: - [ ] I suggest not disabling the linters. Linters ensure you use best practices in programming. Try resolving linter errors manually

File: src/redux/Dragon/Dragon.js [hunk 3]
```diff
@@ -16,7 +15,7 @@ export const fetchDragons = createAsyncThunk(
       id: dragon.id,
       name: dragon.name,
       type: dragon.type,
-      flickr_images: dragon.flickr_images,
+      img: dragon.flickr_images[0],
```
Comment (shella12) [inline src/redux/Dragon/Dragon.js:18]: Great job @mrsamsharma  🤝

- [ ] There seems to be an error with the way axion is imported. I think you need to install axion in `Dragon.js`.

error

**Review by shella12 — CHANGES_REQUESTED** [review]
##
## Hi @mrsamsharma  ✋

Good job so far!

 You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next step of the project. You are almost there!

almost-there

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**Review by soesandarwin2201 — APPROVED** [review]
Hi @mrsamsharma , 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Congrats

## Good Highlights✔️
- Linter checks are passing
- Good commit messages
- Adhered to Gitflow



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**